tester.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
Please move this file to `a/unit-tests` dir and add info about it in README.org.